### ISSUE TITLE
 Could not load file or assembly protobuf-net

### ISSUE THREAD

**Issue by skarpecki** [ddrinka/ApacheOrcDotNet#8]
I cloned the repo, and had tried to run either WriterTest or ReaderTest but I got following exception:

` Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'protobuf-net, Version=3.0.0.0, Culture=neutral, PublicKeyToken=257b51d87d2e4d67' or one of its dependencies. The system cannot find the file specified.`

I am using VS 2019, tried to restore nuget packages, uninstall and install them all, clean/rebuild/build solution, nothing seems to help. I also tried to create new project that references ApacheOrcDotNet base project, just to open ORC file - still the same. I honestly have no idea what might wrong - maybe VS version? Which version you use that works fine?

I've tried to use library on 3 different machines (well actually two physical, but one with two Windows 10 instances).

Log file from Assembly Binding Log Viewer (fuslovw) attached:
[fuslogvw_log_protobuf.txt](https://github.com/ddrinka/ApacheOrcDotNet/files/6812362/fuslogvw_log_protobuf.txt)

**Comment by ddrinka** [comment]
Hi @skarpecki, thanks for all the diagnostics you went through trying to track this down.

I pushed a change to master to correct the framework versions of each project.  I hadn't tested the `.App` projects after the previous change, so that may have broken them.

I also created a GitHub Actions test to see if any errors would reproduce.  It looks clean on my end: https://github.com/ddrinka/ApacheOrcDotNet/runs/3061932958

I use Visual Studio 2019, but I'd recommend testing with the dotnet CLI tools as the GitHub Action workflow does, to see if your behavior is any different.
